database/mysql.go

```diff
@@ -26,6 +27,7 @@ type MySQL struct {
 	password    string
 	dumpCommand string
 	dumpPath    string
+	extraArgs   string
```

Comment:
Change name to `additional_options`

https://github.com/backup/backup/blob/master/lib/backup/database/mysql.rb#L34

Reply:
Renamed